test/dlc_tests/sanity/test_security_check.py

```diff
@@ -53,6 +55,10 @@ def test_ecr_scan(image, ecr_client):
     :param image: str Image URI for image to be tested
     :param ecr_client: boto3 Client for ECR
     """
+    current_account_id = sts_client.get_caller_identity().get("Account")
+    image_account_id = get_account_id_from_image_uri(image)
+    if image_account_id != current_account_id:
+        image = _reupload_image_to_test_ecr(image, ecr_client, region, current_account_id)
```

Comment:
how is this getting image from production which is released

Reply:
We check the URI for the image to be tested and find the `image_account_id`. We then compare it to the `current_account_id` for the job that is executing this test.

If they are different, we execute the `_reupload_image_to_test_ecr()` helper function which uploads the image to the account where the test is being run, therefore making it possible to run the ECR scan on the image.